test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1707,31 +1703,18 @@ BOOST_AUTO_TEST_CASE(transfer_ether)
 				throw;
 			}
 		}
-
-		contract D {
-			// This takes about ~3600 gas, which exceeds the 2300 gas stipend.
-			function () payable {
-				bytes32 tmp = 1;
-				for (uint i = 0; i < 20; i++)
-					tmp = sha3(tmp);
```

Comment:
Actually this example has a somewhat legitimate use for the `.gas()` option. The recipient could do non-state changing, but expensive operations to determine the sender (imagine a token contract).